src/xds_ascii.cpp

```diff
@@ -53,4 +53,21 @@ void XdsAscii::apply_polarization_correction(double p, Vec3 normal) {
   }
 }
 
+/// \par overload is maximally allowed pixel value in a peak (MAXC).
+void XdsAscii::eliminate_overloads(double overload, size_t nover) {
+
+  size_t n1 = data.size();
+  auto it = data.begin();
+  while (it != data.end()) {
+    if ((*it).maxc > overload) {
```

Comment:
It would depend on `overload` value, but I suppose MAXC would usually not exceed OVERLOAD value? (e.g. in 16 bit CCD case and if OVERLOAD=65535).

Reply:
I thought `maxc >= overload` would be better and consistent with XDS, but probably I was wrong.

Reply:
The XDS documentation has the following on [OVERLOAD](https://xds.mr.mpg.de/html_doc/xds_parameters.html#OVERLOAD=)
"... The maximum value of this proportional range is specified by the mandatory input parameter OVERLOAD=. If a pixel contents exceeds this maximum value the pixel is overloaded; a reflection is overloaded if it includes one or more overloaded pixels. ", so I think the code is correct.

Reply:
Yes, thanks.